stacks/satnav-stack.js

```diff
@@ -22,7 +23,9 @@ export function SatnavStack({ stack }) {
   const stackConfig = getConfig(stack.stage)
 
   // Get carpark reference
-  const { carparkBucket, carparkEventBus } = use(CarparkStack)
+  const { carparkBucket } = use(CarparkStack)
```

Comment:
I wonder if we can limit the permissions here so that the index writer can only _read_ files from the bucket.

Reply:
We should look into overall permissions yes. I will create issue